src/quo2/components/info/information_box.cljs

```diff
@@ -56,7 +56,9 @@
   [{:keys [type closable? closed? id icon style button-label on-button-press on-close]} message]
   (let [background-color (get-color-by-type type :bg)
         border-color     (get-color-by-type type :border)
-        icon-color       (get-color-by-type type :icon)
+        icon-color       (case (name icon)
+                           "untrustworthy" :no-color
+                           :none (get-color-by-type type :icon))
```

Comment:
Please can you elaborate why these changes are needed.

Reply:
Because otherwise the white content of the `untrustworthy` icon (exclamation mark) would get painted over.

I decided to specifically do this check in `information-box` component, because there shouldn't be the need to manually set something like `:no-color true` on the top level component and pass it down to the icon all the way.

What do you think?

Reply:
Thank you for explanation. There might be other use cases so let's not hard-code for `untrustworthy`
Maybe we can do something like this
```
icon-color (if no-icon-color? :transparent (get-color-by-type type :icon))
```
And we can remove changes from info_message.cljs

Reply:
Done! 👍

Reply:
`:transparent` doesnt' quite work as an icon color. It will fallback to `(colors/theme-colors colors/neutral-100 colors/white)`

https://github.com/chrispader/status-mobile/blob/893fc2c6db5101f653c2733edaf86482c83f8a45/src/quo2/components/icon.cljs#L23

Reply:
I'd instead just pass the `:no-icon-color?` prop to `info-message` and then the icon

Reply:
![Screenshot 2022-10-20 at 15 41 10](https://user-images.githubusercontent.com/20173411/196964695-10a140e7-4fa2-4b95-8ff5-bf4b7cdbd215.png)